client/src/components/LandingPage/Header.js

```diff
@@ -150,7 +150,7 @@ class Header extends PureComponent {
               <Link to="/login">Log in</Link>
             </li>
             <li className="links">
-              <Link to="/register">Sign up</Link>
+              <Link to="/login">Log in</Link>
```

Comment:
Why do we have two "Log in" options in the navbar now?

Reply:
I thought I did the opposite